### ISSUE TITLE
Command line improvements

### ISSUE THREAD

**Issue by xsawyerx** [PerlDancer/Dancer2#1111]
This meta ticket is here to collect ideas for improvements to the command line that comes with Dancer2 (`dancer2`). We can discuss ideas here and create appropriate tickets for them.

**Comment by tushardave26** [comment]
Nowadays, almost every web developer uses one of the popular front-end framework such as Bootstrap, Foundation, Materialize and many more in their project pipeline. Dancer2 command-line scripts just works fine out of the box but if someone uses the front-end frameworks for their work, it become really a cumbersome process to change all the default HTML templates or change their content. So I would love to see that feature in Dancer2 CLI as option "-f frameworkName" which provides the HTML templates according to selected front-end framework. For the first initial release of this option can accommodate most popular front-end framework Bootstrap, Foundation and Materialize. If users will like the option, we can add more front-end frameworks.

I can help core team to add this feature but I am not sure from where to start. Please guide me so I can also be useful to the community.

Thanks.

**Comment by xsawyerx** [comment]
I think this would best be handled by "profiles" that allow you to pick from a set of common configurations (one using Bootstrap, Foundation, and Materialize, another using FontAwesome).

Having the program handle all the combination of these is probably not likely to happen. That would be too difficult.

Does that make sense?

**Comment by tushardave26** [comment]
Yes. But how can I accomodate profiles into my Dancer2 app? Is there any automatic way of doing it?

**Comment by xsawyerx** [comment]
No ability right now, but we're here collecting ideas and stuff to work on. :)

I'll add this as another idea and open a ticket for it. Sounds good?

**Comment by tushardave26** [comment]
Thanks Sawyer. I appreciate that. 

Please let me know if you or core developer team need any help. 

I have never got chance to thank you and entire Daner team for your great work. Thanks so much for this beautiful and easy to learn framework.

**Comment by xsawyerx** [comment]
It's all worthwhile knowing people enjoy it. :)
